src/scss/attendee/theme.scss

```
@@ -15,9 +15,11 @@
   --panel-background: rgb(255, 255, 255);
   --glass-panel-background: rgba(255, 255, 255, 0.4);
   --color-glass-panel-background: rgba(99, 42, 199, 0.7);
+  --color-glass-panel-background-hover: #632ac7;
```

let's keep all the variables in the same style?
```suggestion
  --color-glass-panel-background-hover: rgb(99, 41, 199);
```

---

[81823af](https://github.com/sparkletown/sparkle/pull/3008/commits/81823afa93b14face968a41f2014dc43a98e655b)